extensions/api/api-core/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/contractdefinition/model/Transformer.java

```diff
@@ -0,0 +1,8 @@
+package org.eclipse.dataspaceconnector.api.datamanagement.contractdefinition.model;
```

Comment:
Probably not part of this PR

Reply:
already deleted. was a copy-paste error.